common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -344,10 +344,10 @@ main() {
             ;;
         "status")
             status_function
+            return $?
             ;;
         "roll_logs")
             roll_logs_function
-	    return $?
```

Comment:
Shouldn't this return remain?

Reply:
I've no idea. 
@morantr , was the script supposed to return immediately here after the roll_logs_function? Or was your intention to still have the chmod call to be made?

Reply:
This return seems useless to me, `roll_logs_function` doesn't have any return. `$?` will have exit code of last executed command from `roll_logs_function` if i am not mistaken.
Also, this is indirect check.